Server/tests/integration/test_manage_gameobject_look_at.py

```
@@ -0,0 +1,77 @@
+@pytest.mark.asyncio
+async def test_look_at_without_target_still_sends(monkeypatch):
+    """look_at without look_at_target should still send the command (C# will error)."""
+    captured = {}
+
+    async def fake_send(cmd, params, **kwargs):
+        captured["params"] = params
+        return {"success": False, "message": "look_at_target is required"}
+
+    monkeypatch.setattr(manage_go_mod, "async_send_command_with_retry", fake_send)
+
+    resp = await manage_go_mod.manage_gameobject(
```

Ruff (0.15.2)

[error] Local variable `resp` is assigned to but never used

Remove assignment to unused variable `resp`

(F841)